tests/qprogram/test_calibration.py

```diff
@@ -0,0 +1,110 @@
+import os
+
+import pytest
+
+from qililab.qprogram.calibration import Calibration
+from qililab.waveforms import Arbitrary, FlatTop, Gaussian, IQPair, Square
+from qililab.yaml import yaml
+
+
+class TestCalibration:
+    """Unit tests checking the Calibration attributes and methods."""
+
+    def test_init(self):
+        """Test init method"""
+        calibration = Calibration()
+
+        assert isinstance(calibration, Calibration)
+        assert isinstance(calibration.operations, dict)
+        assert len(calibration.operations) == 0
+
+    def test_add_operation(self):
+        """Test add_operation method"""
+        xpi = Square(1.0, 100)
```

Comment:
In the tests should we try other options or only to check that is works in the basics?